_posts/2021-02-08-pulp-squeezer-0.0.7.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Automating Pulp Debian workflows with Pulp Squeezer
+author: Melanie Corr, Matthias Dellweg
+tags:
+  - release
+---
+
+We are happy to announce the release of Pulp Squeezer 0.0.7!
+
+With the [Pulp 3 Debian](https://docs.pulpproject.org/pulp_deb/installation.html) plugin, you can take full control over the management and distribution of your Debian and Ubuntu packages. With this release of [Pulp Squeezer](https://galaxy.ansible.com/pulp/squeezer), you can automate many aspects of the Pulp Debian workflow using Ansible.
+
+[Pulp Squeezer](https://galaxy.ansible.com/pulp/squeezer) is a growing collection of Ansible modules that you can use to manage Pulp.
+
+As part of this release, you can now use Pulp Squeezer to automate workflows with the following new modules:
+
+* `deb_distribution`
+* `deb_publication`
+* `deb_remote`
+* `deb_repository`
+* `deb_sync`
+
+For the `deb_sync` module, you might want to register the sync as a [handler](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html).
```

Comment:
Ok, let's pump this up a little. ;) Having an ansible article without the word "idempotent" feels rummy.

```suggestion
The first four in the list, associated with actual resources, provide the `state` parameter to allow for idempotent playbooks the ansible way.
The `deb_sync` module however, being an action module, you might want to register as a [handler](https://docs.ansible.com/ansible/latest/user_guide/playbooks_handlers.html).
```